pkg/inject/inject.go

```diff
@@ -1023,25 +1059,34 @@ func (conf *ResourceConfig) populateMeta(obj runtime.Object) error {
 		conf.pod.spec = &v.Spec
 		conf.pod.meta = &v.ObjectMeta
 
-		if conf.ownerRetriever != nil {
-			kind, name, err := conf.ownerRetriever(v)
-			if err != nil {
-				return err
-			}
-			conf.workload.ownerRef = &metav1.OwnerReference{Kind: kind, Name: name}
```

Comment:
This change makes it so that ownerRef is no longer set, but it is still used in controller/proxy-injector/webhook.go#Inject